package.json

```diff
@@ -22,7 +23,7 @@
     "@skyux/i18n": "^4.0.0 || ^5.0.0-alpha.0",
     "@skyux/indicators": "^4.0.0 || ^5.0.0-alpha.0",
     "@skyux/layout": "^4.0.0 || ^5.0.0-alpha.0",
-    "angular-tree-component": "^8.5.6"
+    "@circlon/angular-tree-component": "^11.0.3"
```

Comment:
I wonder if since we are supporting Angular 9 in this if we also need to support the tree component's version 9 that supports Angular 9 since the newest version only supports 11+. Maybe something to flex on.

Reply:
Good catch. Downgraded to the Angular 9 version.